features/fixtures/app/lib/scenarios/native_project_packages_scenario.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:MazeRunner/channels.dart';
+import 'package:bugsnag_flutter/bugsnag_flutter.dart';
+
+import 'scenario.dart';
+
+class NativeProjectPackagesScenario extends Scenario {
+  @override
+  Future<void> run() async {
+    await MazeRunnerChannels.runScenario('NativeProjectPackagesScenario', arguments: {
+      'notifyEndpoint': endpoints.notify,
+      'sessionEndpoint': endpoints.sessions
+    });
+
+    await bugsnag.attach(
+      runApp: () async {
```

Comment:
Not sure I understand why this `async` is here?

I *think* this could also be expressed as:
```suggestion
      runApp: () => throw Exception('Keep calm and carry on'),
```